docs/RevPi-HAT-EEPROM-Format.md

```diff
@@ -305,6 +308,24 @@ This file represents the (first) MAC address for this device. If the device has
 ##### Example(s) <!-- omit in toc -->
 `C8:3E:A7:01:32:5E`
 
+#### (6) EEPROM Data Version
+
+This file represents the version of the content of the EEPROM. The version must be incremented every time a new version of the revpi-hat-data for the product is released. So the EEPROM Data Version is independent for each product.
```

Comment:
Suggestion:
'This field/attribute represents the EEPROM content version. It mus be incremented every time a new version of ...'

You should at least change 'This file' to 'This field' or 'This attribute'. I would prefer 'attribute', as it is used above to describe the table contents. Or was it intentional to use the word file? If not, the subsection MAC Address also has the word file.

Reply:
It was intentional, as the values are presented as files to the "user".

Reply:
But as other attributes use the work attribute, it would be better to be consistent

Reply:
Just remove it and say something like "It represents...". Should be clear enough for the readers.